samples/public/BlankUwpNet9App/App.xaml.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.Storage;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+namespace BlankUwpNet9App
+{
+    /// <summary>
+    /// Provides application-specific behavior to supplement the default Application class.
+    /// </summary>
+    sealed partial class App : Application
+    {
+        /// <summary>
+        /// Initializes the singleton application object. This is the first line of authored code
+        /// executed, and as such is the logical equivalent of main() or WinMain().
+        /// </summary>
+        public App()
+        {
+            InitializeComponent();
+
+            Suspending += OnSuspending;
+        }
+
+        /// <inheritdoc/>
+        protected override void OnLaunched(LaunchActivatedEventArgs e)
+        {
+            Debugger.Launch();
```

Comment:
Is this intended to be pushed? Or was it for local debugging purposes only?

Reply:
Debug only. Removed.